string robot_name
---
float32 robot_charge
